packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2550,17 +2473,12 @@ export class StripeHelper {
    * only change in their entirety.
    */
   async processPaymentMethodEventToFirestore(event: Stripe.Event) {
-    const { data } = event;
-
-    const paymentMethod = data.object as Stripe.PaymentMethod;
-
-    if (event.type === 'payment_method.detached') {
-      await this.stripeFirestore.removePaymentMethodRecord(paymentMethod.id!);
```

Comment:
Why don't we delete anymore?

Reply:
Because if we want to check for card fingerprints used in the future, it can be useful to know who previously used the card.